src/briefcase/bootstraps/pursuedpybear.py

```diff
@@ -0,0 +1,150 @@
+from briefcase.bootstraps.base import BaseGuiBootstrap
+
+
+class PursuedPyBearGuiBootstrap(BaseGuiBootstrap):
+    def app_source(self):
+        return """\
+import importlib.metadata
```

Comment:
To ensure this is clear....the app source for all these third-party GUI toolkits were doing a special import for `import.metdata` that pivoted on Python 3.8. Since Briefcase only supports >=3.8 now, I changed it just to import `importlib.metadata` always now.